src/components/Pagination.tsx

```diff
@@ -0,0 +1,145 @@
+import classNames from 'classnames';
+import { FC, useState } from 'react';
+
+const perPageOptions = [3, 5, 10, 20];
+
+interface Props {
+  total: number,
+  page: number,
+  setPage: (page: number) => void,
+  withInfo: true | undefined,
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  page,
+  setPage,
+  withInfo,
```

Comment:
to make it true by default
```suggestion
  withInfo = true,
```